torch/linalg/__init__.py

```diff
@@ -961,12 +962,17 @@
 
 Args:
     A (Tensor): tensor of shape `(*, m, n)` where `*` is zero or more batch dimensions.
-    tol (float, Tensor, optional): the tolerance value. See above for the value it takes when `None`.
-                                   Default: `None`.
-    hermitian(bool, optional): indicates whether :attr:`A` is Hermitian if complex
-                               or symmetric if real. Default: `False`.
+    tol (float, Tensor, optional): [NumPy compatibility] the tolerance value.
+                                   If the value of :attr:`tol` is `None`
+                                   then it's equivalent to setting `rtol=tol` and `atol=tol` otherwise.
```

Comment:
What about the simpler:
```suggestion
    tol (float, Tensor, optional): [NumPy Compat] Alias for :attr:`atol`. Default: `None`.
```
If it's specified to `None` or not specified then this function does what it normally does by default, right?

Reply:
Right, I applied your suggestion.